addons/crm_claim/crm_claim.py

```diff
@@ -42,7 +43,7 @@ class crm_claim(osv.osv):
 
     def _get_default_stage_id(self, cr, uid, context=None):
         """ Gives default stage_id """
-        team_id = self.pool['crm.team']._get_default_team_id(cr, uid, context=context)
+        team_id = self.pool['crm.team']._get_default_team_id(cr, SUPERUSER_ID, context=context)
```

Comment:
are you sure? the sales team is based on the current user (`['|', ('user_id', '=', user_id), ('member_ids', 'in', user_id)]`) so accessing it as SUPERUSER will returns you the wrong team